dev/gallery.tsx

```diff
@@ -0,0 +1,130 @@
+/* eslint monorepo/no-internal-import: "off", monorepo/no-relative-import: "off", import/no-relative-packages: "off" */
```

Comment:
**idea**: I wonder if we should just update our root `.eslintrc` to ban the `monorepo/*` rules for this `dev` folder? Might make for less noise in the code here. 🤷‍♂️